### PR TITLE
Recommendation system

### PR DESCRIPTION
**Pull request by RMayeux**

### CODE REVIEW

**Review by clemcx — APPROVED** [review]
okok

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function index(
                     break;
             }
         } else {
-            $demands = $demandRepository ->findBy(array('status' => array(0, 1)), array('deadline' => 'ASC'));
```
Comment (benj-san) [inline src/Controller/AdminController.php:67]: Can you change the array() by [] to be in sync with new PSR norms (array() are not used anymore)

File: src/Controller/AdminController.php [hunk 2]
```diff
@@ -112,11 +112,59 @@ public function advisor(AdvisorRepository $advisorRepository, EntityManagerInter
...
+        AdvisorRepository $advisorRepository,
+        Board $board,
+        Request $request,
+        DemandRepository $demandRepository
+    ): Response {
         $advisor = $advisorRepository->findAll();
+        $demand = $demandRepository->findOneBy(['id' => $board->getDemand()]);
+        $tags = $demand->getTags()->getValues();
+
+        $advisorsArray = array();
```
Comment (benj-san) [inline src/Controller/AdminController.php]: []

**Review by benj-san — COMMENTED** [review]
Clean, just transform the array() into [] and I'll approve it

File: src/Controller/AdminController.php (hunk 1 above)
Comment (RMayeux) [reply src/Controller/AdminController.php:67]: Fixed

File: src/Controller/AdminController.php (hunk 2 above)
Comment (RMayeux) [reply src/Controller/AdminController.php]: Fixed

**Review by benj-san — APPROVED** [review]

**Pull request merged**